src/utils/plotUtils.js

```diff
@@ -105,4 +133,6 @@ export {
   colorInterpolator,
   hexToRgb,
   convertRange,
+  generateCellIdMapping,
+  reorderObjectValues,
```

Comment:
I see that these two functions are only used in Embedding.jsx. In this case, it is better to have them in there, rather than in a different file.